iothub_client/samples/iothub_client_sample_upload_to_blob_mb/iothub_client_sample_upload_to_blob_mb_stress.c

```diff
@@ -54,6 +58,7 @@ static IOTHUB_CLIENT_FILE_UPLOAD_GET_DATA_RESULT getDataCallback(IOTHUB_CLIENT_F
                 *data = (const unsigned char*)data_to_upload;
                 *size = block_size;
                 blocks_sent++;
+                (void)printf("."); fflush(stdout);
             }
```

Comment:
Oh wow. We have a stress test in the samples directory  now? hmm.

Reply:
This branch was required by a customer that had problems with stress. We need to seriously think about a stress-test solution.
For now, this is a compromise for the uhttp_blob branch.